tests/testsuite/plugins.rs

```diff
@@ -94,7 +94,7 @@ fn plugin_to_the_max() {
         .file("src/lib.rs", "pub fn baz() -> i32 { 1 }")
         .build();
 
-    foo.cargo("build").run();
+    foo.cargo("check").run();
```

Comment:
I would avoid any changes to `plugins.rs`, since these involve linking and such.